R/rest.R

```diff
@@ -114,7 +114,7 @@ call_mlflow_api <- function(..., client, query = NULL, data = NULL, verb = "GET"
           data,
           pretty = TRUE,
           auto_unbox = TRUE,
-          na = NULL
+          na = "null"
```

Comment:
ah this is probably my bad. i think i had added the `na = NULL`

Reply:
all good, I don't think this actually affected anything and tbh it's weird that your approach didn't work